eras/shelley/impl/src/Cardano/Ledger/Shelley/API/Types.hs

```diff
@@ -50,6 +50,7 @@ import Cardano.Ledger.Keys as X (
   KeyRole (..),
   SignKeyDSIGN,
   SignKeyKES,
+  UnsoundPureSignKeyKES,
```

Comment:
Unsound stuff should not be exported from the library, because it is only suitable for testing
```suggestion
```

Reply:
I'll remove it, because it doesn't really need to be exported here, but unfortunately it is not just used for testing.
The UnsoundPure API is basically the old, non-mlocked KES API; we need to keep it around for consensus to load KES sign keys from disk. This is inherently unsound, and we want to phase it out, but until we do, the unsound API is going to be used in production, just like it has been (under a less abrasive name) since the introduction of KES.

Reply:
> The UnsoundPure API is basically the old, non-mlocked KES API; we need to keep it around for consensus to load KES sign keys from disk.

I understand this. But there is no need to re-export this unsound serialization from other APIs if it will be removed in the near future.

Reply:
Agree. Which is why I've removed it.